gobblin-core/build.gradle

```diff
@@ -57,6 +57,7 @@ dependencies {
   compile externalDependency.oltu
   compile externalDependency.opencsv
   compile externalDependency.hadoopHdfs
+  compile externalDependency.parquet
```

Comment:
We plan to decouple external extension from gobblin-core. Please make a new module, say `gobblin-parquet` in `gobblin-modules` and move the new classes there, similar to what we do to `gobblin-sql`.

In this way, an extension can be brought in on-demand instead of bundled with `gobblin-core` and brought in regardless of whether the application needs it or not.